modules/sr/robot/randomizer.py

```diff
@@ -21,3 +24,21 @@ def add_jitter(
     random_range = actual_value * (random_range_percent / float(100))
     new_value = actual_value + random.uniform(-random_range, random_range)
     return type(actual_value)(max(min_possible, min(new_value, max_possible)))
+
+
+def add_independent_jitter(
+    actual_value: T,
+    min_possible: T,
+    max_possible: T,
+    std_dev_percent: float =  # % of full scale value
+    DEFAULT_RANDOM_INDEPENDENT_RANGE_PERCENT,
```

Comment:
Should this be on the same line? Reads weirdly this way

```suggestion
    std_dev_percent: float = DEFAULT_RANDOM_INDEPENDENT_RANGE_PERCENT,  # % of full scale value
```

Reply:
It was a line length issue but the current label fits (just).